style.css

```diff
@@ -0,0 +1,828 @@
+    display: grid;
+    align-content: center;
+    justify-self: end;
+}
+
+.post-to-twitter-text {
+    font-weight: bold;
+    text-decoration: none;
+    padding: 8px 20px;
+    background-color: dodgerblue;
+    border-radius: 20px;
+}
+
+.post-to-twitter-text:hover {
+    background-color: rgb(0, 79, 158);
+}
+
+/* MAIN FEED */
+
+/* GENERIC POST ADJUSTMENTS */
+
+.main-feed > div {
+    display: grid;
+    grid-template-columns: 50px 1fr;
+    grid-auto-rows: auto;
+    padding: 15px 15px;
+    border-bottom: solid 1px var(--border-color);
+    border-left: solid  1px var(--border-color);
+    border-right: solid  1px var(--border-color);
+
+}
+
+.user-photo-image {
+    object-fit: cover;
+    width: 40px;
+    border-radius: 200px;
+}
+
+.tweet-content {
+    display: grid;
+    grid-template-rows: 20px 1fr 30px;
+    gap: 5px;
+}
+
+.feed-user-details {
+    display: grid;
+    grid-template-columns: 1fr auto;
+}
+
+.feed-user-details > .feed-username {
+    font-size: 14px;
+}
+
+.feed-user-details > .feed-username > span {
+    color: gray;
+}
+
+.feed-user-menu {
+    display: grid;
+    place-self: center;
+    text-align: center;
+    width: 20px;
+}
+
+.single-tweet-content {
+    display: grid;
+    grid-auto-rows: min-content;
+    gap: 10px;
+}
+
+.tweet-photo {
+    display: grid;
+    padding: 5px 0;
+    place-self: center;
+}
+
+.feed-tweet-actions {
+    display: grid;
+    grid-template-columns: 1fr 30px;
+    gap: 30px;
+    align-items: center;
+}
+
+.direct-tweet-reaction {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+}
+
+.direct-tweet-reaction > a {
+    display: grid;
+    grid-template-columns: 30px auto;
+    align-items: center;
+}
+
+.direct-tweet-reaction > a > div {
+    display: grid;
+    align-items: center;
+    color: gray;
+    font-size: 12px;
+    width: fit-content;
+}
+
+.direct-tweet-reaction > a > div > img {
+    width: 30px;
+    padding: 4px;
+    border-radius: 30px;
+}
+
+/* REPLY REACTIONS */
+.reply-reactions {
+    text-decoration: none;
+}
+
+.reply-reactions:hover .reply {
+    filter: invert(41%) sepia(46%) saturate(2810%) hue-rotate(195deg) brightness(103%) contrast(102%);
+}
+
+.reply-reactions:hover > div:nth-of-type(1) {
+    border-radius: 30px;
+    background-color: rgba(30, 143, 255, 0.274);
+}
+
+.reply-reactions:hover > .reply-count {
+    color: dodgerblue;
+}
+
+/* REPOST REACTIONS */
+.repost-reaction {
+    text-decoration: none;
+}
+
+.repost-reaction:hover .repost {
+    filter: invert(46%) sepia(89%) saturate(1171%) hue-rotate(88deg) brightness(94%) contrast(101%);
+}
+
+.repost-reaction:hover > div:nth-of-type(1) {
+    border-radius: 30px;
+    background-color: #02a01445;
+}
+
+.repost-reaction:hover .repost-count {
+    color: #03ba18;
+}
+
+/* LIKE REACTIONS */
+.like-reaction {
+    text-decoration: none;
+}
+
+.like-reaction:hover .like {
+    filter: invert(20%) sepia(62%) saturate(7458%) hue-rotate(318deg) brightness(85%) contrast(100%);
```

Comment:
The use of filters is super powerful and its cool to see how many of the features youve used